targets/furi_hal_include/furi_hal_usb_ccid.h

```diff
@@ -17,12 +17,12 @@ typedef struct {
 } FuriHalUsbCcidConfig;
 
 typedef struct {
-    void (*icc_power_on_callback)(uint8_t* dataBlock, uint32_t* dataBlockLen, void* context);
+    void (*icc_power_on_callback)(uint8_t* dataBlock, uint8_t* dataBlockLen, void* context);
     void (*xfr_datablock_callback)(
         const uint8_t* pcToReaderDataBlock,
-        uint32_t pcToReaderDataBlockLen,
+        uint8_t pcToReaderDataBlockLen,
```

Comment:
On arm32 smaller size types are not going save space or make code faster. Any specific reasons why you want to have uint8_t here?

Reply:
I can change this function back to uint32_t then, since it will return only values up to 255 and it is called only by core components

Reply:
I'm wondering why original structures were designed to have `unit32_t`? Do standard got payloads bigger than 256 bytes?

Reply:
That's because CCID has two modes: Short APDU and Extended APDU.

In Short APDU mode, packets have size up to 0xFF. 
In Extended APDU mode, packets have size up to 0xFFFF

Althouth devices can have different modes, there is only one property in the structure, which can go up to 0xFF or 0xFFFF in either case

Reply:
I'd keep uint32_t, but final decision is up to you.

Reply:
I'll keep it as uint32_t, since:

- the CCID buffer size  is already 255 in size
- we are already checking ii dwLength is < 255
- i will add extended apdu support at some point anyway